setup.py

```diff
@@ -0,0 +1,39 @@
+import os
+from setuptools import find_packages, setup
+
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as readme:
+    README = readme.read()
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='wagtail-contact',
+    version='0.1',
+    packages=find_packages(),
+    include_package_data=True,
+    license='BSD License',  # example license
+    description='A simple Wagtail contact app.',
+    long_description=README,
```

Comment:
This irks me a bit - where did you get this pattern from?

Reply:
Sorry, the whole setup.py or how I read the README.md file?
I just made a few changes to example given here: https://docs.djangoproject.com/en/1.11/intro/reusable-apps/

Reply:
How you read the README file - but apparently it seems to be quite common. Leave it as is for now.